### PR TITLE
add year graph to filter

### PR DESCRIPTION
**Pull request by TylorShin**

### CODE REVIEW

File: app/components/yearGraph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const useStyles = require('isomorphic-style-loader/useStyles');
+
+const YearGraph: FC = () => {
+  useStyles(s);
+  const [currentYearSet, setCurrentYearSet] = useState<Year | null>(null);
+
+  const allYears = useSelector((state: AppState) => state.searchFilterState.yearAll);
+
+  const maxWidth = 180; // px
+  const maxHeight = 100; // px
```
Comment (YooJeongWoo) [inline app/components/yearGraph/index.tsx]: How about pass these variables as props and set a default value?

File: app/components/yearGraph/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  return (
+    <div className={s.wrapper} style={{ width: maxWidth, margin: '0 auto' }}>
+      <svg width={maxWidth} height={maxHeight} role="img">
+        {rects}
+      </svg>
+      <div className={s.labelWrapper}>
+        {currentYearSet && (
+          <div className={s.currentLabelWrapper}>
+            <div className={s.labelText}>{`year: ${currentYearSet.year}`}</div>
+            <div className={s.labelText}>{`count: ${formatNumber(currentYearSet.docCount)}`}</div>
```
Comment (YooJeongWoo) [inline app/components/yearGraph/index.tsx]: 'Number of document' Label will be better than `count:`

Comment (TylorShin) [reply app/components/yearGraph/index.tsx]: I think it's too long for the area

File: app/components/yearGraph/index.tsx (hunk 1 above)
Comment (TylorShin) [reply app/components/yearGraph/index.tsx]: good point. I forgot about it. that was first idea.

File: app/components/yearFilterDropdown/index.tsx [hunk 3]
```diff
@@ -196,6 +197,9 @@ const YearFilterDropdown: React.FC<Props> = React.memo(({ location, history }) =
             value={maxYear}
           />
         </div>
+        <div style={{ margin: '24px 0' }}>
```
Comment (NohYujeong) [inline app/components/yearFilterDropdown/index.tsx]: Why use inline style?

File: app/components/yearGraph/index.tsx (hunk 2 above)
Comment (YooJeongWoo) [reply app/components/yearGraph/index.tsx]: Than `papers:` will be fine!

File: app/components/yearFilterDropdown/index.tsx (hunk 3 above)
Comment (TylorShin) [reply app/components/yearFilterDropdown/index.tsx]: I was tired.

**Pull request merged**